account_invoice_import_simple_pdf/i18n/account_invoice_import_simple_pdf.pot

```diff
@@ -436,6 +436,78 @@ msgstr ""
 msgid "Missing fixed char."
 msgstr ""
 
+#. module: account_invoice_import_simple_pdf
```

Comment:
POT files should be update by https://translation.odoo-community.org/

Reply:
@simahawk  
Agreed if you now, how to call the bot to do the job, he's supposed to do ;-) 

There is no logical change here we've just add a few case for existing invoice numbers, we've encountered in real life.
That's offering a better security in invoice number detection.